spec/dummy/app/themes/default/views/thredded/shared/_top_nav.html.erb

```diff
@@ -1,34 +1,32 @@
-<nav role="navigation" class="main-nav">
+<nav role="navigation" class="user-navigation">
   <ul>
-    <li class="main-nav-menu">
-      <a href="#">
-        <svg viewBox="0 0 32 32"><use xlink:href="#menu"></use></svg>
-        <span>Menu</span>
-      </a>
-    </li>
-    <li class="main-nav-home">
-      <a href="#">
-        <svg viewBox="0 0 32 32"><use xlink:href="#home-2"></use></svg>
-        <span>Home</span>
-      </a>
+    <!-- always -->
+    <li class="user-navigation--search">
+      <!-- <svg viewBox="0 0 32 32"><use xlink:href="#search1"></use></svg> -->
+      <%= render 'thredded/search/form' %>
     </li>
-    <li class="main-nav-session-preferences">
+
+    <!-- if logged in -->
+    <li class="">
       <a href="#">
-        <svg viewBox="0 0 32 32"><use xlink:href="#user1"></use></svg>
+        <!-- <svg viewBox="0 0 32 32"><use xlink:href="#user1"></use></svg> -->
         <span>Sign In / Register</span>
       </a>
     </li>
-    <li class="main-nav-notification-preferences">
-      <a href="#">
-        <svg viewBox="0 0 32 32"><use xlink:href="#bell"></use></svg>
-        <span>Notification Settings</span>
-      </a>
-    </li>
-    <li class="main-nav-search">
-      <a href="#search">
-        <svg viewBox="0 0 32 32"><use xlink:href="#search1"></use></svg>
-        <span>Search</span>
-      </a>
-    </li>
+
+    <!-- if not logged in -->
+    <!-- <li class="main&#45;nav&#45;notification&#45;preferences"> -->
+    <!--   <a href="#"> -->
+    <!--     <!&#45;&#45; <svg viewBox="0 0 32 32"><use xlink:href="#bell"></use></svg> &#45;&#45;> -->
+    <!--     <span>Notification Settings</span> -->
+    <!--   </a> -->
+    <!-- </li> -->
+    <!--  -->
+    <!-- <li class="main&#45;nav&#45;notification&#45;preferences"> -->
+    <!--   <a href="#"> -->
+    <!--     <!&#45;&#45; <svg viewBox="0 0 32 32"><use xlink:href="#bell"></use></svg> &#45;&#45;> -->
+    <!--     <span>UserName</span> -->
+    <!--   </a> -->
+    <!-- </li> -->
```

Comment:
Let me know if I can help break these out with some partials to make the logged in / logged out states. Just give me the word, son!